### PR TITLE
Add frontpage fixture data for initial 4

### LINKED ISSUE #1546: Add fixture data for WS frontpages for initial 4 services and route
**Is your feature request related to a problem? Please describe.**
Add fixture data for the 4 core services and enable it to be requested locally

**Describe the solution you'd like**
Add live data for the services `Pidgin, Igbo, Yoruba and Thai` as fuxture data in simorgh.

Put this data in the folder structure `data/prod/service/frontpage/index.json`
Put this data in the folder structure `data/test/service/frontpage/index.json`

Using a folder with `index.json`, rather than `frontpage.json` allows for more data to be added at a later date for use in storybook.

Route this data in simorgh so that when you request `localhost:7080/service.json` the data is returned. 

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Unit tests not needed. Can be tested locally by hitting 
http://localhost:7080/pidgin.json
http://localhost:7080/igbo.json
http://localhost:7080/yoruba.json
http://localhost:7080/thai.json

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by andrew-nowak**
Resolves #1546 

**Overall change:** Imports frontpage fixture data to /data

**Code changes:**

- Imports fixture data from ares for the ~~hausa~~ thai, igbo, pidgin and yoruba services front pages
- Sets up routing for fixture data when running locally
- Adds placeholder service configs for four services
- Ignore `frontpage` directories in data validator

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/routes/index.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ export const articleRegexPath = `/:service(${serviceRegex})/articles/:id(${idReg
 
 export const articleDataRegexPath = `${articleRegexPath}.json`;
 
+export const frontpageRegexPath = `/:service(${serviceRegex})`;
```
Comment (dr3) [inline src/app/routes/index.js]: I know this doesnt particularly matter for fixture, but I think were going to continue using `.amp` for our frontpages too :)

File: src/server/index.jsx [hunk 2]
```diff
@@ -61,6 +65,36 @@ server
  */
 
 if (process.env.APP_ENV === 'local') {
+  const errorWhenAccessingFile = async dataFilePath => {
```
Comment (dr3) [inline src/server/index.jsx]: As nice as this is, as this is a dev route I dont think we need to be this safe or include a new dependancy. The following should be fine IMO

```
res.sendFile(path, {}, error => {
  if (error) {
    logger.error(error
    res.status(500).send('Unable to find file.');
  }
});
```

**Review by dr3 — CHANGES_REQUESTED** [review]

**Review by dr3 — APPROVED** [review]

**Review by chris-hinds — APPROVED** [review]
LGTM 👍

**Review by sareh — APPROVED** [review]
👍 Looks good locally!

**Comment by jamesbrumpton** [comment]
Happy for this to be merged.

**Pull request merged**